CMakeLists.txt

```diff
@@ -2605,12 +2605,12 @@ find_package(LibUSB)
 # USB HID controller support
 option(HID "USB HID controller support" ON)
 if(HID)
-  find_package(hidapi)
-  # hidapi_VERSION is only available starting with 0.10.0
-  if(NOT hidapi_FOUND OR NOT hidapi_VERSION OR hidapi_VERSION VERSION_LESS 0.10.1)
+  find_package(hidapi 0.10.1)
```

Comment:
What was wrong with this expression above?
Find_package(hidapi 0.10.1) succeeds on Ubuntu Bionic, because it is no version number set.

Reply:
Oof, why does CMake let this succeed...

Reply:
I fixed this by adding `hidapi_VERSION` to the `find_package_handle_standard_args` call.

Reply:
I think the error makes it clear what is happening:
```
-- Could NOT find hidapi (missing: hidapi_VERSION) (Required is at least version "0.10.1")
-- Linking internal libhidapi statically
```